src/microbe_stage/BioProcess.cs

```diff
@@ -75,6 +75,23 @@ public void Check(string name)
         TranslationHelper.CopyTranslateTemplatesToTranslateSource(this);
     }
 
+    public float GetRateWithConditions(BiomeConditions conditions)
```

Comment:
Isn't this very similar to some of the methods in the ProcessSystem? For example `CalculateProcessMaximumSpeed` which has summary comment starting with: `Calculates the maximum speed a process can run at in a biome based on the environmental compounds.`

If that method is too much a simpler variant should be written in the process system class and notes added so that both variants are remembered to be updated at once if future changes is needed for this logic.

Reply:
Was this comment of mine addressed by some commit?

Reply:
I forgot to delete this section, but rate uses the function you suggested now.

Reply:
Should be easy to then make a small commit to revert this part of the PR then, right?